acspy/acsc.py

```diff
@@ -277,27 +300,20 @@ def toPointM(hcomm, flags, axes, target, wait=SYNCHRONOUS):
     call_acsc(acs.acsc_ToPointM, hcomm, flags, axes_c, target_c, wait)
 
 
-def enable(hcomm, axis, wait=SYNCHRONOUS):
+def enableMotor(hcomm, axis:int, wait=SYNCHRONOUS):
```

Comment:
We shouldn't rename this in case others are depending on it (like some of my apps!) You can create an alias if you'd like, but I've just been mirroring the ACS function names, which here is just `enable`.

Reply:
Hi Pete, ok! On it. Thanks for your answer.